src/applications/static-pages/facilities/FacilityPatientSatisfactionScoresWidget.jsx

```diff
@@ -5,109 +5,104 @@ import { displayPercent } from '../../../platform/utilities/ui';
 import FacilityApiAlert from './FacilityApiAlert';
 import { connect } from 'react-redux';
 
-export class FacilityPatientSatisfactionScoresWidget extends React.Component {
-  render() {
-    if (this.props.loading || !Object.keys(this.props.facility).length) {
-      return (
-        <LoadingIndicator message="Loading facility patient satisfaction scores..." />
-      );
-    }
+export function FacilityPatientSatisfactionScoresWidget(props) {
```

Comment:
I didn't destructure `props` because the function name was so long that prettier would force the args onto new lines.